mcstatus/pinger.py

```diff
@@ -33,6 +34,8 @@
     },
 }
 
+FIND_MINECRAFT_COLORS_REGEX = re.compile(r"[\xA7|&][0-9A-FK-OR]", re.IGNORECASE)
```

Comment:
```suggestion
DESCRIPTION_COLORS_RE = re.compile(r"[\xA7|&][0-9A-FK-OR]", re.IGNORECASE)
```
Also change all references to this, and including imports of this constant in other files.

Reply:
Renamed via force push.